test failure looks real:

```
ERROR: corehq.util.workbook_reading.tests.test_errors:SpreadsheetErrorsTest.test_file_not_found(<function open_any_workbook at 0x7f480d7a9578>, u'xls')

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 338, in test

    return test_func(self, *args)

  File "/mnt/commcare-hq-ro/corehq/util/workbook_reading/tests/test_errors.py", line 24, in test_file_not_found

    with open_workbook(get_file('nosuchfile', ext)):

  File "/usr/local/lib/python2.7/contextlib.py", line 17, in __enter__

    return self.gen.next()

  File "/mnt/commcare-hq-ro/corehq/util/workbook_reading/adapters/generic.py", line 12, in open_any_workbook

    with open_xls_workbook(filename) as workbook:

  File "/usr/local/lib/python2.7/contextlib.py", line 17, in __enter__

    return self.gen.next()

  File "/mnt/commcare-hq-ro/corehq/util/workbook_reading/adapters/xls.py", line 23, in open_xls_workbook

    except xlrd.CompDocError as e:

AttributeError: 'module' object has no attribute 'CompDocError'
```

--- a/corehq/util/workbook_reading/adapters/xls.py
+++ b/corehq/util/workbook_reading/adapters/xls.py
@@ -20,7 +20,7 @@ def open_xls_workbook(filename):
             raise SpreadsheetFileEncrypted(six.text_type(e))
         else:
             raise SpreadsheetFileInvalidError(six.text_type(e))
-    except xlrd.CompDocError as e:
+    except xlrd.compdoc.CompDocError as e:
         raise SpreadsheetFileInvalidError(six.text_type(e))
     except IOError as e:
         raise SpreadsheetFileNotFound(six.text_type(e))

